File: apps/multimodal-screener/iOS/Aiberry/UI/Theme.swift
import SwiftUI

/// Warm, editorial "private notes" palette shared with the VoiceVitals demo:
/// cream background, serif headlines, soft rounded cards, sage-green icon tiles,
/// deep-green accent. No brand colors — the design language matches `iOS/VoiceVitals`.
enum Theme {
    static let bg       = Color(red: 0.953, green: 0.945, blue: 0.918)  // warm cream
    static let card     = Color(red: 0.912, green: 0.903, blue: 0.872)  // soft warm-gray card
    static let cardAlt  = Color(red: 0.972, green: 0.966, blue: 0.945)  // lighter inner fill

    static let ink      = Color(red: 0.118, green: 0.118, blue: 0.106)  // near-black text
    static let inkSoft  = Color(red: 0.435, green: 0.427, blue: 0.400)  // muted secondary text

    static let tile     = Color(red: 0.792, green: 0.851, blue: 0.733)  // sage icon tile
    static let tileInk  = Color(red: 0.310, green: 0.447, blue: 0.282)  // deep green glyph
    static let dark     = Color(red: 0.168, green: 0.165, blue: 0.149)  // charcoal pill

    // Semantic aliases.
    static let accent   = tileInk        // primary action / selection
    static let positive = tileInk
    static let warn     = Color(red: 0.85, green: 0.62, blue: 0.26)
    static let danger   = Color(red: 0.83, green: 0.36, blue: 0.33)

    static let corner: CGFloat = 22
}

extension Font {
    /// Editorial serif display font.
    static func serif(_ size: CGFloat, _ weight: Font.Weight = .regular) -> Font {
        .system(size: size, weight: weight, design: .serif)
    }
}

extension View {
    /// Standard soft "card" container.
    func card(_ padding: CGFloat = 18) -> some View {
        self
            .padding(padding)
            .frame(maxWidth: .infinity, alignment: .leading)
            .background(Theme.card)
            .clipShape(RoundedRectangle(cornerRadius: Theme.corner, style: .continuous))
    }
}

File: apps/Voice-Biomarker/Android/app/src/main/java/ai/zetic/voicevitals/ui/Theme.kt
package ai.zetic.voicevitals.ui

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.fillMaxWidth
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.text.font.FontFamily
import androidx.compose.ui.unit.Dp
import androidx.compose.ui.unit.dp

/**
 * Warm, editorial "private notes" palette: cream background, serif headlines,
 * soft rounded cards, sage-green icon tiles, charcoal pill buttons.
 */
object Theme {
    val Bg = Color(0.953f, 0.945f, 0.918f)   // warm cream
    val Card = Color(0.912f, 0.903f, 0.872f) // soft warm-gray card
    val CardAlt = Color(0.972f, 0.966f, 0.945f) // lighter inner fill

    val Ink = Color(0.118f, 0.118f, 0.106f)  // near-black text
    val InkSoft = Color(0.435f, 0.427f, 0.400f) // muted secondary text

    val Tile = Color(0.792f, 0.851f, 0.733f) // sage icon tile
    val TileInk = Color(0.310f, 0.447f, 0.282f) // deep green glyph
    val Dark = Color(0.168f, 0.165f, 0.149f) // charcoal pill
    val Lavender = Color(0.792f, 0.749f, 0.925f) // avatar accent

    // Semantic aliases.
    val Accent = TileInk
    val Positive = TileInk
    val Warn = Color(0.85f, 0.62f, 0.26f)
    val Danger = Color(0.83f, 0.36f, 0.33f)

    val Corner: Dp = 22.dp
    val Serif = FontFamily.Serif
}

/** Standard soft "card" container. */
fun Modifier.card(padding: Dp = 18.dp): Modifier = this
    .fillMaxWidth()
    .clip(RoundedCornerShape(Theme.Corner))
    .background(Theme.Card)
    .padding(padding)
